core/google/cloud/iterator.py

```diff
@@ -175,12 +220,23 @@ def next(self):
 class Iterator(object):
     """A generic class for iterating through Cloud JSON APIs list responses.
 
-    Sub-classes need to over-write :attr:`ITEMS_KEY` and to define
-    :meth:`_item_to_value`.
-
     :type client: :class:`~google.cloud.client.Client`
     :param client: The client, which owns a connection to make requests.
 
+    :type path: str
+    :param path: The path to query for the list of items. Defaults
+                 to :attr:`PATH` on the current iterator class.
+
+    :type items_key: str
+    :param items_key: The key used to grab retrieved items from an API
+                      response. Defaults to :data:`DEFAULT_ITEMS_KEY`.
```

Comment:
I take this to mean you'd like me to update the docstring to say "(Optional) The key..."?